docs/overview/51-telemetry-observability-overview.md

```diff
@@ -3,15 +3,13 @@ title: Telemetry and Observability
 type: Observability
 ---
 
-Out of the box, Kyma provides tools to collect and expose **telemetry** data, such as metrics, traces, and log data.
-
-Of course, you'll want to view and analyse the data you're collecting. This is where **observability** tools come in. 
+Out of the box, Kyma provides tools to collect and expose **telemetry** data, such as metrics, traces, and log data. Of course, you'll want to view and analyse the data you're collecting. This is where **observability** tools come in. 
 
 ## Collecting data
 
-The [OpenTelemetry](https://opentelemetry.io/) observability framework is the core tool where all of Kyma's raw data comes together. 
+The [OpenTelemetry](https://opentelemetry.io/) observability framework is the core tool where all of Kyma's raw data comes together.  Among the data sources that flow into the OpenTelemetry collector are Kubernetes and Istio.
 
-Among the data sources that flow into the OpenTelemetry collector are Kubernetes and Istio.
+> **Note** All metrics relevant for observing the in-cluster Istio Service Mesh are collected separately. You can find more information about it on [Istio monitoring](link-to-istio-monitoring).
```

Comment:
You can find more information about it on [Istio monitoring](link-to-istio-monitoring). -> **in** Istio monitoring **(documentation**) ?

Reply:
good point, thanks!